practice_05.18.19_polymorphysm/Task4_Calculator/Calculator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Threading;
+
+namespace practice_05._18._19_polymorphysm.Task4_Calculator
+{
+    public class Calculator
+    {
+        private Operation[] operations =
+        {
```

Comment:
Please use constructor for initialization fields with default values.

Reply:
fixed